<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>596</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>801</width>
    <height>596</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>801</width>
    <height>596</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton{
	border-radius: 8px;
	background-color: rgb(207, 218, 255);
}
QPushButton:hover{
	color: white;
	background-color: rgb(85, 170, 127);
}
QPushButton:focus{
	outline:none;
}

</string>
    </property>
    <property name="text">
     <string>NỘP BÀI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(84, 136, 237);</string>
    </property>
    <property name="text">
     <string>Bộ đề :  Bài thực hành mở</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>600</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 219, 255);</string>
    </property>
    <property name="text">
     <string>Số fibonacci lớn hơn k</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 8px;
	background-color: rgb(207, 218, 255);
}
QPushButton:hover{
	color: white;
	background-color: rgb(255, 3, 62);
}
QPushButton:focus{
	outline:none;
}</string>
    </property>
    <property name="text">
     <string>THOÁT</string>
    </property>
   </widget>
   <widget class="QLabel" name="classDisplay">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(84, 136, 237);</string>
    </property>
    <property name="text">
     <string>Lớp : 62TH-NB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>540</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 219, 255);</string>
    </property>
    <property name="text">
     <string>NỘP</string>
    </property>
   </widget>
   <widget class="QLabel" name="markDisplay">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(84, 136, 237);</string>
    </property>
    <property name="text">
     <string>Điểm tổng : 0 / 15000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 8px;
	background-color: rgb(207, 218, 255);
}
QPushButton:hover{
	color: white;
	background-color: rgb(85, 170, 127);
}
QPushButton:focus{
	outline:none;
}

</string>
    </property>
    <property name="text">
     <string>XEM ĐỀ BÀI</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameDisplay">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(84, 136, 237);</string>
    </property>
    <property name="text">
     <string>Họ và tên: Nguyễn Tuấn Vũ</string>
    </property>
   </widget>
   <widget class="QTableView" name="Info_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(84, 136, 237);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>560</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 219, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="sjDisplay">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(84, 136, 237);</string>
    </property>
    <property name="text">
     <string>Môn học :  Thuật toán ứng dụng</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>590</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 219, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="idDisplay">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(84, 136, 237);</string>
    </property>
    <property name="text">
     <string>Mã sinh viên : 2051063451</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>741</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_2{
	border: 1.5px solid black;
	border-radius: 8px;
}</string>
    </property>
    <widget class="QTableWidget" name="titleList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>721</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10.5pt &quot;;
border: 0px solid #ccc;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="statusScroll">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>571</width>
      <height>151</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>569</width>
       <height>149</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>571</width>
      <height>23</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>Info_table</zorder>
   <zorder>submitButton</zorder>
   <zorder>label_6</zorder>
   <zorder>label_18</zorder>
   <zorder>exitButton</zorder>
   <zorder>classDisplay</zorder>
   <zorder>label_11</zorder>
   <zorder>markDisplay</zorder>
   <zorder>viewButton</zorder>
   <zorder>nameDisplay</zorder>
   <zorder>label_16</zorder>
   <zorder>sjDisplay</zorder>
   <zorder>label_17</zorder>
   <zorder>idDisplay</zorder>
   <zorder>widget_2</zorder>
   <zorder>statusScroll</zorder>
   <zorder>progressBar</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
